src/components/about-us/opening/anchors-panel.js

```diff
@@ -61,14 +63,13 @@ const TopRow = styled.div`
   }
   ${screen.mobile`
     height: ${headerStyle.height.mobile};
-    padding: ${headerStyle.padding.mobile};
+		padding-left: 40px;
```

Comment:
indent?

Reply:
Minor. Just a thought for the coding style problem(e.g., indentation).
Would we want to include the indentation validation within in the CI validation?
This can reduce the effort for the style check during review.

Reply:
The most style problems are fixed by `npm run lint` before `git commit`.
However, this one is currently not checked by the eslint. We need to find eslint rules or plugins to check `template strings`.